http-clients/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     compile libs.wsRsApi
 
     testCompile project(':http-servers')
+    testCompile files('lib/VersionTest.jar')
```

Comment:
regex for the version?

On Tue, Jun 21, 2016 at 2:33 PM ellisjoe <notifications@github.com> wrote:

> In http-clients/build.gradle
> <https://github.com/palantir/http-remoting/pull/137#discussion_r67955852>:
>
> > @@ -21,6 +21,7 @@ dependencies {
> >      compile libs.wsRsApi
> >
> >      testCompile project(':http-servers')
> > +    testCompile files('lib/VersionTest.jar')
>
> I'd have to update the test any time the version changes.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/137/files/03ef9531ab351411ef1ec59f745ed66d74da5aa2#r67955852>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGOdwUAg9pHreJJubEGgNbQj2KlCYvtGks5qOFhSgaJpZM4I7JKz>
> .
>